app/models/atomic_lti/platform.rb

```diff
@@ -1,5 +1,8 @@
 module AtomicLti
   class Platform < ApplicationRecord
     validates :iss, presence: true
```

Comment:
Should this have a query_constraint of ISS?  We're treating it as a primary key.